catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -293,28 +305,88 @@ void registryInfoQuery() throws UnsupportedQueryException {
         SourceResponse identityMetacard = query(queryRequest);
         if (identityMetacard.getResults()
                 .size() > 0) {
-            remoteName = identityMetacard.getResults()
+            String metacardTitle = identityMetacard.getResults()
                     .get(0)
                     .getMetacard()
                     .getTitle();
+            setId(metacardTitle);
             registryId = RegistryUtility.getRegistryId(identityMetacard.getResults()
                     .get(0)
                     .getMetacard());
+            updateConfiguration(metacardTitle);
         }
-        updateConfiguration();
+
     }
 
-    private void updateConfiguration() {
+    private String getObjectClassDefinition(Bundle bundle, String pid) {
```

Comment:
This method should be renamed since it doesn't return the ObjectClassDefinition anymore